src/focus-ring.js

```diff
@@ -29,22 +31,31 @@ document.addEventListener('DOMContentLoaded', function() {
 	    }
 	    return triggers;
 	},
+        _addClass = function(el, htmlClass) {
+            el.className += ' '+htmlClass;   
+        },
+        _removeClass = function(el, htmlClass) {
```

Comment:
Do you know the support level of RegExp in older IE? It might be nicer to do something like:

```
this.element.className = this.element.className.replace(regExp(name), '');
```

based on the polyfill example in https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Polyfill